src/core/sys/linux/unistd.d

```diff
@@ -18,3 +19,730 @@ enum {
 
 /// Prompt for a password without echoing it.
 char* getpass(const(char)* prompt);
+
+// linux system call number from Linux's asm/unistd.h
+version(X86)
+{
+    enum SYS_restart_syscall = 0;
```

Comment:
`enum SYS: clong { restart_syscall = 0; ... }` is a bit too much (for the C's API)?

Reply:
I would prefer this as well! I just wasn't sure about this and there seemed to be a couple of old precedents on `SYS_` when I did this (one year ago).

Reply:
I think it's fair to use enums here, but that's just my preference. It's easy to generate global enums from it at ctfe anyway, if it's required.